File: Solutions/OnlineSites/CodeChef/Practice/Beginner/Id and Ship/FLOW010.py
T = int(input(""))
 
i = 0
 
while(i < T):
    ship = input("").lower()
    
    if(ship == 'b'):
        print("BattleShip")
    elif(ship == "c"):
        print("Cruiser")
    elif(ship == "d"):
        print("Destroyer")
    elif(ship == "f"):
        print("Frigate")
 
    i = i + 1 

File: Solutions/OnlineSites/CodeChef/Practice/Beginner/Id and Ship/FLOW010.cpp

#include <iostream>
#include <ctime>
#include <math.h>
#include <string>
#include <iomanip>
#include <vector>

int main(int argc, char ** args) {

	unsigned long t;

	std::cin >> t;

	for(int i = 0; i < t; i++){
		char c;

		std::cin >> c;

		switch(c){
		case 'b':
		case 'B':
			std::cout << "BattleShip" << std::endl;
			break;
		case 'c':
		case 'C':
			std::cout << "Cruiser" << std::endl;
			break;
		case 'd':
		case 'D':
			std::cout << "Destroyer" << std::endl;
			break;
		case 'f':
		case 'F':
			std::cout << "Frigate" << std::endl;
			break;
		default:
			break;
		}
	}

	return 0;
}
